runtime/services/catalog/migrations.go

```diff
@@ -12,10 +13,14 @@ import (
 	"github.com/rilldata/rill/runtime/pkg/dag"
 	"github.com/rilldata/rill/runtime/pkg/fileutil"
 	"github.com/rilldata/rill/runtime/services/catalog/artifacts"
+
+	// Register some standard stuff.
 	_ "github.com/rilldata/rill/runtime/services/catalog/artifacts/sql"
 	_ "github.com/rilldata/rill/runtime/services/catalog/artifacts/yaml"
 	"github.com/rilldata/rill/runtime/services/catalog/migrator"
-	_ "github.com/rilldata/rill/runtime/services/catalog/migrator/metrics_views"
+
+	// Register some standard stuff.
+	_ "github.com/rilldata/rill/runtime/services/catalog/migrator/metricsviews"
 	_ "github.com/rilldata/rill/runtime/services/catalog/migrator/models"
 	_ "github.com/rilldata/rill/runtime/services/catalog/migrator/sources"
```

Comment:
There's a non blank import in the group that should be moved up. Also, add actual descriptions here (e.g. `// Load migrators`)